themes/cakephp/static/cake.css

```diff
@@ -337,6 +337,18 @@ a.improve {
 #sidebar-navigation > ul > li:lang(fr):nth-of-type(1):before {
     content: "Préface";
 }
+#sidebar-navigation > ul > li:lang(es):nth-of-type(1):before {
```

Comment:
Just to encourage people to translate these titles, but I can remove these if necessary